yt_dlp/extractor/facebook.py

```diff
@@ -427,6 +428,42 @@ def _perform_login(self, username, password):
             self.report_warning('unable to log in: %s' % error_to_compat_str(err))
             return
 
+    def _get_video_metadata(self, url):
+        ffmpeg = FFmpegPostProcessor()
```

Comment:
The extractor should not start downloading the file through FFmpeg/FFprobe to extract video metadata.
This is *maybe* something the core library could do (behind a command line argument), though even there I am not convinced.